app/views/static_pages/front_page.html.erb

```diff
@@ -1,38 +1,139 @@
+                <a href="https://www.meetup.com/shescoding-eastside/">Redmond/Bellevue Meetup</a>
+              </li>
+            </ul>
+          </section>
+          <section class="section site-section">
+              <h2>Resources</h2>
+              <p>Find educational and informational content aimed toward women in tech (or those interested in tech), our allies, and
+                companies looking to learn more about diversity and inclusion.</p>
+              <div class="text-center mt-5">
+                <a class="cta" href="/resources">Resources</a>
+              </div>
+
+              <!-- We will add resources here later.-->
+              <!--
+              <h3>Latest in Resources:</h3>
+              <ul class="featured-list">
+                <li><a href="#">link to page</a> in <a href="#">category page</a></li>
+                <li><a href="#">link to page</a> in <a href="#">category page</a></li>
+                <li><a href="#">link to page</a> in <a href="#">category page</a></li>
+                <li><a href="#">link to page</a> in <a href="#">category page</a></li>
+                <li><a href="#">link to page</a> in <a href="#">category page</a></li>
+              </ul>
+              -->
+            </section>
+            <section class="section site-section">
+                <h2>Mentorship</h2>
+                <p>We are super thrilled about the launch of the She's Coding Mentorship Program. For a while now we thought about how
+                  to best share this network of amazing technical women, minorities and allies that we are building. This mentorship
+                  program is our first attempt into a direction of direct enablement and empowerment through matching based on their
+                  career goals, their background, their interests and their expertise.</p>
+                <div class="text-center mt-5">
+                  <a class="cta" href="/mentorship">Join Us</a>
                 </div>
-            </div>
+            </section>
+            <section class="section site-section as-seen-on green-links">
+                <div class="description">
+                <h2>As Seen On
+                  <cite>"CODE: Debugging the Gender Gap"</cite>
+                </h2>
+                <p>CODE documentary exposes the dearth of American female and minority software engineers and explores the reasons for
+                  this gender gap. CODE raises the question: what would society gain from having more women and minorities code?</p>
+                <a href="https://www.codedoc.co/" target="_blank">View the documentary</a>
+              </div>
+              <div class="iframe-container">
+                <iframe src="https://player.vimeo.com/video/136884902" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+              </div>
+            </section>
         </div>
     </div>
-</div>
+  </main>
+
+  <%= render :partial => "layouts/footer" %>
```

Comment:
I added the footer in this PR. You should see it if pull changes and run locally.

Reply:
Yup, I saw it - thanks Anya! :D